waspc/cli/src/Wasp/Cli/Command/WaspLS.hs

```diff
@@ -23,8 +23,8 @@ data Args = Args
   }
 
 data Options = Options
-  { optionsLogFile :: Maybe FilePath,
-    optionsUseStdio :: Bool
+  { _optionsLogFile :: Maybe FilePath,
+    _optionsUseStdio :: Bool
```

Comment:
Fixes the warning @shayneczyzewski mentioned on slack.

Reply:
Got it!
How is it that you prefixed both with _ though?